<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1418</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>0</y>
      <width>161</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_13">
     <item row="1" column="0">
      <widget class="QTreeWidget" name="TreeProjet">
       <column>
        <property name="text">
         <string>Liste des Projets</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Projet 1</string>
        </property>
        <item>
         <property name="text">
          <string>TacheMulti1</string>
         </property>
         <item>
          <property name="text">
           <string>Tache1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tache2</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>TacheMulti2</string>
         </property>
         <item>
          <property name="text">
           <string>Tache3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tache4</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Tache5</string>
         </property>
        </item>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_43">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Vos Projets &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>0</y>
      <width>491</width>
      <height>452</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_14">
     <item row="4" column="0">
      <widget class="QTableWidget" name="EmploiDuT"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Veuillez sélectionner une date&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Programme du jour sélectionné&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCalendarWidget" name="Calendrier"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Votre Agenda&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>417</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="6">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="2" colspan="6">
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="titreTache">
       <property name="text">
        <string>titre</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="2">
      <widget class="QDateEdit" name="echeanceTache">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2015</year>
         <month>6</month>
         <day>2</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Titre</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Si Unitaire,veuillez remplir les informations sivantes&lt;/p&gt;&lt;p&gt;Sinon, cliquez sur ajouter Tache&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Date de disponibilité</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="5">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Associer une(des) Tache(s) précédente(s)</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1" colspan="5">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Duree</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QDateEdit" name="dispoTache">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2015</year>
         <month>6</month>
         <day>1</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QCheckBox" name="Preempt">
       <property name="text">
        <string>Preemptive</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Projet associé</string>
       </property>
      </widget>
     </item>
     <item row="21" column="0" colspan="3">
      <widget class="QPushButton" name="AjouterTache">
       <property name="text">
        <string>Ajouter Tache !</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="5">
      <widget class="QTableWidget" name="TachePrec">
       <column>
        <property name="text">
         <string>Titre </string>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Ajouter une Tache&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QRadioButton" name="unitaire">
       <property name="text">
        <string>Unitaire</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QSpinBox" name="DureeTache"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Date d'échéance</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QComboBox" name="ProjetList">
       <item>
        <property name="text">
         <string>------</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Projet1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Projet2</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QRadioButton" name="composite">
       <property name="text">
        <string>Composite</string>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Ajouter à une tache composite ? </string>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QComboBox" name="TacheCompoList"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Type de la tache</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLineEdit" name="DecrTacheUni">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>463</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>410</y>
      <width>175</width>
      <height>185</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Créer un nouveau projet&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="NomProjet">
       <property name="text">
        <string>Nom du projet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NomProjet_2">
       <property name="text">
        <string>Nom du Projet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Date Dispo </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="DatedispoProj">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2015</year>
         <month>6</month>
         <day>1</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Echeance</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="EcheanceProj">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2015</year>
         <month>6</month>
         <day>1</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AjouterProjet">
       <property name="text">
        <string>AjouterProjet</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>178</width>
      <height>130</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Programmer une tache&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QLineEdit" name="idtache">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Horaire</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Titre</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Date </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateEdit" name="datetache">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2015</year>
           <month>6</month>
           <day>1</day>
          </datetime>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTimeEdit" name="heuretache"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="ProgrammeTache">
       <property name="text">
        <string>Programmer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;OU&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>237</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Programmer un événement &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>271</y>
      <width>196</width>
      <height>199</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_34">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Programmer une Activité &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLineEdit" name="DescripActivite">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>Duree</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="DureeActivite"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_35">
       <property name="text">
        <string>Lieu</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="LieuActivite">
       <property name="text">
        <string>Lieu</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Date </string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QDateEdit" name="dateActi">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2015</year>
         <month>6</month>
         <day>1</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Horaire</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QTimeEdit" name="HeureActi"/>
     </item>
     <item row="7" column="1" colspan="2">
      <widget class="QPushButton" name="AjouterActivite">
       <property name="text">
        <string>Programmer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>line_10</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>AjouterProjet</zorder>
   <zorder>NomProjet</zorder>
   <zorder>NomProjet_2</zorder>
   <zorder>label_12</zorder>
   <zorder>DatedispoProj</zorder>
   <zorder>label_14</zorder>
   <zorder>EcheanceProj</zorder>
   <zorder>label_9</zorder>
   <zorder>NomProjet</zorder>
   <zorder>label_16</zorder>
   <zorder>label_18</zorder>
   <zorder>dateActi</zorder>
   <zorder>HeureActi</zorder>
   <zorder>DecrTacheUni</zorder>
   <zorder>label_35</zorder>
   <zorder>LieuActivite</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1418</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
